app/BeaconBridge/Controllers/SubmissionController.cs

```diff
@@ -0,0 +1,64 @@
+using BeaconBridge.Constants.Submission;
+using BeaconBridge.Data;
+using BeaconBridge.Data.Entities.Submission;
+using BeaconBridge.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace BeaconBridge.Controllers;
+
+[ApiController, Route("api/[controller]")]
+public class SubmissionController(SubmissionContext submissionContext, SubmissionStatusService statusService,
+  UserHelper userHelper, ILogger logger) : ControllerBase
+{
+  /// <summary>
+  /// Update the status of a submission to a TRE.
+  /// </summary>
+  /// <param name="subId">The ID of the submission to update.</param>
+  /// <param name="statusType">The new status for the submission.</param>
+  /// <param name="description">A description to accompany the update.</param>
+  /// <returns></returns>
+  [Authorize(Roles = "dare-control-admin,dare-tre-admin")]
+  [HttpGet("UpdateStatusForTre")]
+  [SwaggerResponse(statusCode: 200, description: "The submission was updated successfully.")]
+  [SwaggerResponse(statusCode: 400, description: "The submission is either closed or non-existent")]
+  public async Task<IActionResult> UpdateStatusForTre(string subId, StatusType statusType, string? description)
+  {
+    try
+    {
+      await UpdateSubmissionStatus(subId, statusType, description);
+      await submissionContext.SaveChangesAsync();
+      return Ok();
+    }
+    catch (Exception e) when (e is DbUpdateException or DbUpdateConcurrencyException or OperationCanceledException)
+    {
+      logger.LogCritical("Unable to save updates for submission {SubId}", subId);
+      return StatusCode(500);
```

Comment:
I think a `50x` is more appropriate as this implies an error on our side. Errors like `DbUpdateException` occur because of something going wrong on our side. Whereas using `409` or any `40x` error is when the user sends us bad info, for example.